### PR TITLE
Added Russian translation of the overview docs page

### LINKED ISSUE #3705: Translate about page to Russian
Related to: #2962 
If you would like to contribute to our translations, please visit the above issue and comment with the language you wish to translate in! (provided it isn't taken yet)

Translate About us page from English to Russian

### PR DESCRIPTION
**Pull request by gulyapulya**
## Issue This PR Addresses

Fixes #3705

Part of  #2962 which combines all translations

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [x] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR adds Russian translation for the overview page of docs.

Examples:

image

image

image

## Steps to test the PR

- cd src\web\docusaurus
- npm install
- npm run start -- --locale ru
- Click 'Docs' button top left corner

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by gulyapulya** [comment]
Hi @sirinoks, could you please review this PR? (you said you can check Russian translations)

Sorry for being late on the issue #3705... I had enough PRs for Hacktoberfest, but came back for it now.  :)

**Comment by IvaniGabrovsky** [comment]
I got a Russan speaker to review the Russian language. Everything looks good and ready to be merged.

**Comment by gulyapulya** [comment]
Thanks, @IvaniGabrovsky for the review!

**Comment by gulyapulya** [comment]
@sirinoks I guess your review is optional now, but still would be great to have 😊 Please let us know if you will be available.

File: src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md [hunk 1]
```diff
@@ -1,120 +1,101 @@
...
+## В Поисках Новой "Планеты"
 
-[Written in Python by Jeff Waugh and Scott James Remnant](https://people.gnome.org/~jdub/bzr/planet/devel/trunk/),
-Planet could be configured with a list of blog feeds and an HTML template. It would
-use these to dynamically generate a site with posts in chronological order from
-the specified feeds.
+Наша "Планета" умирает. Программное обеспечение, которое мы используем, последний раз обновлялось 13 лет назад. В то время как основной его код ушел в прошлое, наши потребности двигают нас вперед. Поддерживать существующую систему, особенно с учетом количества студентов, участвующих в Open Sourse в Seneca, стало слишком сложно. Наш нынешний сайт часто ломается и нуждается в ручном вмешательстве на регулярной основе. В будущем нам нужна новая "Планета", которую мы могли бы назвать домом.
 
-## In Search of a New Planet
+Готовясь войти в 2020 год, мы решили, что пора рассмотреть вопрос перехода на новую систему. К сожалению, почти каждая приходящая на смену Planet система сама становилась такой, что не может быть обслуживаемой.
```
Comment (sirinoks) [inline src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md:35]: > система сама становилась такой, что не может быть обслуживаемой

система становилась необслуживаемой/устаревшей/забытой/заброшенной

File: src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md [hunk 2]
```diff
@@ -1,120 +1,101 @@
...
+- Переработка CI/CD в GitHub Actions
+- Улучшено SEO
+- Добавление Firebase в качестве внутреннего хранилища данных для информации о пользователе
+- Улучшения аутентификации на основе SAML, авторизации JWT и безопасности процессов регистрации пользователей
+- Новый дизайн пользовательского интерфейса, логотип, CSS и тематика
+- Улучшенная доступность и пользовательский опыт
+- Миграция монолитной серверной части в микросервисы (завершено на 90 %) и шлюз API с использованием Traefik
+- Улучшения в Elasticsearch и Redis
+- Полный порт интерфейса с GatsbyJS на next.js
+- TypeScript переписывает внешний интерфейс
```
Comment (sirinoks) [inline src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md:89]: > TypeScript rewrite of front-end

Тут имелось ввиду скорее
`Переписка внешнего интерфейса с использованием/с помощью TypeScript`

File: src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md [hunk 3]
```diff
@@ -1,120 +1,101 @@
...
-sidebar_position: 1
+sidebar_position: 0
 ---
 
+import StarField from '@site/src/components/StarField';
+
 # Обзор Telescope
 
-## Introduction
+## Наши вкладчики
```
Comment (sirinoks) [inline src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md:9]: Как-то "вкладчики" не так звучит. Я пыталась найти как обычно переводят contributors в других местах, и нашла одно место [тут](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/main/README-RU.md#open-source-%D0%B8%D0%BD%D0%B8%D1%86%D0%B8%D0%B0%D1%82%D0%B8%D0%B2%D1%8B-%D0%BA%D0%BE%D0%BD%D1%82%D1%80%D0%B8%D0%B1%D1%8C%D1%8E%D1%82%D0%B5%D1%80%D0%BE%D0%B2). Тут просто тупо переводят - контрибьюторы. Не знаю, что из двух лучше - оба варианта как-то не очень. Можно вообще забить, и перевести как "разработчики", или "наша команда". Но так не интересно, хоть, может, и красивее..

Let me know what you think of this

**Review by sirinoks — CHANGES_REQUESTED** [review]
Good job! Mostly my comments are not about errors, but rather things to consider improving. Some are concrete changes/corrections though.

File: src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md [hunk 4]
```diff
@@ -1,120 +1,101 @@
 ---
-sidebar_position: 1
+sidebar_position: 0
```
[the remark this answers is not in the document]
Comment (gulyapulya) [reply src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md:2]: Docusaurus produces an old version of the overview page (without the contributors part in the beginning). So, I copied from the overview itself as was done in the Ukrainian translation. I guess it should be the order of the translations, but I was not sure about that number as different translations either have 0 or 1.

File: src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md [hunk 5]
```diff
@@ -1,120 +1,101 @@
...
-posts of our colleagues, we find that we are not alone in our struggles to make
-things work, our interests in various topics, and that imposter syndrome isn't
-something unique to "me."
+## Что такое Planet?
 
-To better enable the discovery of blogs within our community, we set up an open
-source blog [Planet](<https://en.wikipedia.org/wiki/Planet_(software)>): an aggregated
-feed of blog posts from Seneca faculty and students working on open source in a
-single page. Our blog Planet currently lives at [http://zenit.senecac.on.ca/~chris.tyler/planet/](http://zenit.senecac.on.ca/~chris.tyler/planet/).
+> Planet (перевод:Планета) — это приложение-агрегатор лент, предназначенное для сбора статей из блогов членов интернет-сообщества и отображения их на одной странице. Планета работает на веб-сервере. Он создает страницы с записями из исходных лент в хронологическом порядке, начиная с самых последних записей. --[Wikipedia](<https://en.wikipedia.org/wiki/Planet_(software)>)
```
[the remark this answers is not in the document]
Comment (gulyapulya) [reply src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md:25]: Да, я тоже над этим думала, но я решила отталкиваться от приложение-агрегатор (оно), потому что Планета здесь не переводится, а используется как название. 

Yeah, I was thinking about that too, but I used "оно" intentionally as the most correct form would be to refer to the word it describes which is "приложение-агрегатор".

File: src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md (hunk 3 above)
Comment (gulyapulya) [reply src/web/docusaurus/i18n/ru/docusaurus-plugin-content-docs/current/overview.md:9]: Я вот тоже долго искала как назвать, потому что ни один из вариантов не был на 100 процентов верным, потому я наверно оставлю на англ и сделаю ремарку на перевод.

**Comment by gulyapulya** [comment]
Thank you for the review @sirinoks! It was so great to discuss all these little uncertainties in the translation, as I tried to come up with the best version, but you never know until someone confirms.

**Comment by gulyapulya** [comment]
I have implemented all the changes suggested, @humphd let me know what you think. 😊

**Review by sirinoks — APPROVED** [review]

**Pull request merged**